addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -43,6 +50,19 @@ const UrlPickerUserValueWidget = InputUserValueWidget.extend({
         $(this.inputEl).trigger('input');
         this._onUserValueChange(ev);
     },
+    /**
+     * Redirects to the current url
```

Comment:
- `to the URL the widget currently holds` or something like that. "Redirects to the current url" means reloading the page to me.
- `.`
- `@private`